bundle/manifests/oadp-operator.clusterserviceversion.yaml

```diff
@@ -496,6 +500,18 @@ spec:
     spec:
       clusterPermissions:
       - rules:
+        - apiGroups:
+          - apiextensions.k8s.io
+          resources:
+          - customresourcedefinitions
+          verbs:
+          - '*'
```

Comment:
I worry about this permission. Can we reduce it in some way? Maybe fewer verbs and specific resources?

Reply:
Done, reduced the scope and updated the perms.